packages/core/tsconfig.json

```diff
@@ -1,6 +1,10 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
+    "types": [
+      "mocha",
+      "node"
+    ],
     "outDir": "dist",
     "module": "commonjs",
```

Comment:
I also had to add `generate-hooks` to `includes` so my VS Code could do typechecking, not sure what's the problem

Reply:
Introduced